enclave-manager/web/src/client/enclaveManager/LocalKurtosisClient.ts

```diff
@@ -6,7 +6,7 @@ import { KurtosisClient } from "./KurtosisClient";
 
 export class LocalKurtosisClient extends KurtosisClient {
   constructor() {
-    const defaultUrl = new URL(window.location.href);
+    const defaultUrl = new URL(`${window.location.protocol}//${window.location.host}`);
```

Comment:
@Dartoxian is this actually correct?
1) We need the full url with params (and fragments?) so we can read the package id and parameters etc.
2) I think this url is missing the colon? (moot point given `1)` ).

Reply:
1. I think when running against localhost at least we don't want that. I made this change mainly for my local development - I was finding that on every reload I was served the enclave list, rather than the correctly routed content (and links were broken because of this)
2. I agree this looks weird, but:
![image](https://github.com/kurtosis-tech/kurtosis/assets/4419574/452011bf-b79e-4626-9f62-9f1283016ec4)